worldedit-core/src/main/java/com/sk89q/worldedit/extension/platform/Platform.java

```diff
@@ -206,4 +207,6 @@ default String getId() {
      * @return A set of supported side effects
      */
     Set<SideEffect> getSupportedSideEffects();
+
+    RelighterFactory getRelighterFactory();
```

Comment:
This method needs **detailed** documentation before this PR is approved and merged.